.github/workflows/publish-layoutlib.yml

```diff
@@ -0,0 +1,30 @@
+name: publish-layoutlib
+
+on:
+  workflow_dispatch:
+
+env:
+  GRADLE_OPTS: "-Dorg.gradle.jvmargs=-Xmx4g -Dorg.gradle.daemon=false -Dkotlin.incremental=false"
+
+jobs:
+
+  publish:
+    runs-on: ubuntu-latest
+    if: github.ref =~ 'refs/heads/**/layoutlib**'
```

Comment:
interesting; what's the difference between this and using branch above?

Reply:
Love it!

Reply:
@geoff-powell If it's manual, why is it necessary to put an arbitrary condition on the branch name? Only accounts with write access to the repo can trigger workflow_dispatch (because they need to push a branch in the repo), why can't it be triggered on any branch? Is this maybe a historical remnant from an earlier version where trigger was on tag push?